src/makeCalculator.js

```diff
@@ -5,6 +5,45 @@
  */
 function makeCalculator() {
   // write code here
+  return {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(number) {
+      this.result += number;
+
+      return this;
+    },
+
+    subtract(number) {
+      this.result -= number;
+
+      return this;
+    },
+
+    multiply(number) {
+      this.result *= number;
+
+      return this;
+    },
+
+    divide(number) {
+      this.result /= number;
```

Comment:
You need to add a check for 0, because you cannot divide by 0 according to the rules of mathematics

Reply:
Thanks, solved.